packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/observability/probe/NewConnectionActivatedProbe.kt

```diff
@@ -58,6 +57,7 @@ class NewConnectionActivatedProbe : DatabaseSessionStateListener {
                 isGenuine = serverInfo.isGenuineMongoDb,
                 nonGenuineServerName = serverInfo.nonGenuineVariant,
                 serverOsFamily = serverInfo.buildEnvironment["target_os"],
+                atlasHost = serverInfo.serverUrl!!.hosts[0].takeIf { serverInfo.isAtlas && !serverInfo.isLocalAtlas },
```

Comment:
- What `serverInfo.isAtlas` checks? is it `atlasVersion` || `checking the urls matches atlas url` as we do for other tools? We also don't look for serverInfo.isLocalAtlas.
- Does atlasHost has port in it? I think normally hosts[0] has both `host:port`. We need to send only host in telemetry
- Is serverUrl has resolved URL?

Reply:
It looks like we only check host URLs `val isAtlas = connectionString.hosts.all { it.matches(atlasRegex) }`, when compass also checks for atlasVersion https://github.com/mongodb-js/compass/blob/main/packages/data-service/src/instance-detail-helper.ts#L149

Reply:
Oh, it seems that Compass is doing some additional checks. I've based the logic of this class on: https://github.com/mongodb-js/mongodb-build-info/blob/main/index.js#L5.

Do you know why Compass might do something extra?

Reply:
Discussed in Slack. Here is a short summary for reference.

> The atlasVersion check is the right way to identify atlas, but not all deployments return a version, so as a fallback we use `mongodb-build-info`.

The follow-up ticket [INTELLIJ-37](https://jira.mongodb.org/browse/INTELLIJ-37)